src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -16,7 +16,7 @@ import scala.concurrent.duration._
 
 object Config {
 
-  private val config = ConfigFactory.load().getConfig("etc-client")
+  private val config = ConfigFactory.load().withFallback(ConfigFactory.defaultOverrides()).getConfig("etc-client")
```

Comment:
Is this change (specific line) necessary? The reference.conf in the jar should be picked up by 
`ConfigFactory.load().getConfig("etc-client")`. Was it tried without the 'withFallback'?